torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -2537,6 +2567,57 @@ void ProcessGroupNCCL::broadcastUniqueNCCLID(
   }
 }
 
+void ProcessGroupNCCL::allgatherUniqueNCCLID(
+    int rootRank,
+    ncclUniqueId* ncclID,
+    std::vector<ncclUniqueId>& ncclIDs) {
```

Comment:
nit: `allgatherUniqueNCCLIDs` (s at the end).

nit: document the arg fields.

Reply:
Looks like we don't usually do that in cpp for the arg doc..